homeassistant/components/lg_soundbar/strings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "config": {
+    "step": {
+      "details": {
```

Comment:
Same here I guess?
```suggestion
      "user": {
```

Reply:
thanks, i was using details then change it by request, see https://github.com/home-assistant/core/pull/71153#discussion_r872640411, forgot to rename the magic strings in the json.